### DISCUSSION TITLE
Name поля - "user" и Импорт/Экспорт

### DISCUSSION THREAD

**Discussion by diSWAG** [totumonline/totum-ru-issues-and-discussions#360] [ОШБК!] [answered]
Name поля - "user" не является запрещенным, и собственно при названии какого-либо поля "user" проблем не происходит и все работает как надо. 

Ровно до момента Импорта/Экспорта, экспорт проходит прекрасно, а вот при импорте мы получаем ошибку:

user не может быть name поля. Выберите другой name.
Функция schemaUpdate; Линия кода setSchema.; Функция while; action поле f_upload таблицы ttm__load_updates

<img width="606" alt="Снимок экрана 2023-04-28 в 12 41 41" src="https://user-images.githubusercontent.com/122656882/235114171-a01065b9-11ad-4f57-aada-2d14e1c0cddf.png">

Поправьте пожалуйста ошибку в обновлении.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
ок, посмотрим

**Reply by ttm-alexeypolunin** [reply]
Есть общий запрет на создание поля user (на уровне postgresql) — поясните подробнее, что вы там делали.

**Reply by diSWAG** [reply]
Да если честно, ничего особенного не делали, просто не зная о запрете на создание поля user, насоздавали их в разных табличках в Тотуме, и потом наткнулись при импорте не вышеуказанную ошибку.

В самом Тотуме поле user легко создается и работает, ошибок никаких не выводится.

<img width="1395" alt="Снимок экрана 2023-05-02 в 09 29 33" src="https://user-images.githubusercontent.com/122656882/235595425-2858802a-52f1-421e-bb9c-243806662f51.png">

**Reply by ttm-alexeypolunin** [reply]
Вот непонятно, как оно создалось, тк там блокировка аж двойная. Может в какой-то 2-3 версии ее не было, но это было достаточно давно.

**Reply by ttm-alexeypolunin** [reply]
Обнаружилось, что его можно установить через Изменить name — исправим, чтоб нельзя было.

Что бы через загрузку обрабатывалось не сделаем, сорри

**Reply by diSWAG** [reply]
Да, верно name возникло через переименование. 
Ничего страшного, мы отказываемся от поля с именем name, что-бы вдруг где боком не вылезло.
Спасибо за ответ!
